src/index.ts

```diff
@@ -113,23 +113,24 @@ export type KycStatusResponse = {
 / Fiat Account Endpoint Types
 */
 
-// Path parameters for POST /accounts/:fiatAccountSchema
-export type AddFiatAccountRequestParams = {
-  fiatAccountSchema: FiatAccountSchema
+// Request body for POST /accounts/
+export type PostFiatAccountRequestBody<T extends FiatAccountSchema> = {
```

Comment:
Renamed `Add` to `Post` to stay consistent with http verb, although the sdk and api-starter use `add` extensively, so may not be worth this change. Thoughts?

Reply:
I'm fine with the change, don't feel too strongly